test/model/src/driver/kv_workload_runner_wt.cpp

```diff
@@ -146,6 +290,34 @@ kv_workload_runner_wt::do_operation(const operation::commit_transaction &op)
     return session->session()->commit_transaction(session->session(), config_str.c_str());
```

Comment:
This is a comment on new lines 283-289. I can't put a comment on them directly since they were not changed.
New lines 283-289 do not need to be protected by lock(_connection_lock) line 280 since they do not depend upon any of the code in the critical section and do not access or modify any protected data. They can be moved before line 280.

Check other locks for code like this too.

Reply:
Nice catch! Let me fix that.